src/neo/SmartContract/Native/NativeContract.cs

```diff
@@ -79,6 +81,12 @@ protected NativeContract()
             contractsHashDictionary.Add(Hash, this);
         }
 
+        protected bool CheckCommittees(ApplicationEngine engine)
```

Comment:
Do you prefer this here rather than `CheckPolicy`?

Reply:
What do you mean? I just move it from another class.